core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -416,7 +416,8 @@ object KafkaConfig {
   "start from " + MaxReservedBrokerIdProp + " + 1."
   val MessageMaxBytesDoc = "The maximum message size that the server can receive. Note that this limit also applies " +
     "to the total size of a compressed batch of messages (when compression is enabled). Additionally, in versions " +
-    "0.11 and later, all messages are written as batches and this setting applies to the total size of the batch."
+    "0.11 and later, all messages are written as batches and this setting applies to the total size of the batch. " +
+    "This can be set per topic with the topic level <code>max.message.bytes</code> config."
```

Comment:
I changed this to refer to the `TopicConfig` definition concatenated with the last sentence before merging.